src/block/countdown/edit.js

```diff
@@ -0,0 +1,511 @@
+		if ( attributes.countdownType === 'recurring' ) {
+			setAttributes( { actionOnExpiration: '' } )
+		}
+	}, [ attributes.countdownType ] )
+
+	return (
+		<>
+			{ isSelected && (
+				<>
+					<InspectorTabs />
+					<InspectorStyleControls>
+						<PanelAdvancedSettings
+							title={ __( 'General', i18n ) }
+							id="countdown"
+							initialOpen={ true }
+						>
+							<AdvancedToolbarControl
+								controls={ COUNTDOWN_TYPE_OPTIONS }
+								attribute="countdownType"
+								value={ attributes.countdownType }
+								fullwidth={ true }
+								default={ 'dueDate' }
+								isSmall={ false }
+							/>
+							<span>
+								{ attributes.countdownType === 'dueDate' ? 'End Date' : 'Start Date' }
+							</span>
+							<DateTimePicker
+								currentDate={ attributes.date }
+								is12Hour={ true }
+								onChange={ currentDate => {
+									// Do not include seconds
+									setAttributes( { date: currentDate.slice( 0, currentDate.length - 3 ) } )
+								} }
+								__nextRemoveResetButton={ true }
+							/>
+							{	attributes.countdownType === 'dueDate' && (
+								<>
+									<AdvancedSelectControl
+										label={ __( 'Timezone', i18n ) }
+										value={ attributes.timezone }
+										options={ TIMEZONE_OPTIONS }
+										defaultValue={ '' }
+										attribute="timezone"
+									/>
+									<AdvancedSelectControl
+										label={ __( 'Action on Expiration', i18n ) }
+										value={ attributes.actionOnExpiration }
+										options={ ACTION_ON_EXPIRATION_OPTIONS }
+										defaultValue={ '' }
+										attribute="actionOnExpiration"
+									/>
+									{ attributes.actionOnExpiration === 'showMessage' }
+								</>
+							) }
+							{ attributes.countdownType === 'recurring' && (
+								<Fragment>
+									<AdvancedRangeControl
+										label={ __( 'Days', i18n ) }
+										min={ 0 }
+										max={ 364 }
+										value={ attributes.daysLeft }
+										attribute="daysLeft"
+									/>
+									<AdvancedRangeControl
+										label={ __( 'Hours', i18n ) }
+										min={ 0 }
+										max={ 23 }
+										value={ attributes.hoursLeft }
+										attribute="hoursLeft"
+									/>
+									<AdvancedRangeControl
+										label={ __( 'Minutes', i18n ) }
+										min={ 0 }
+										max={ 59 }
+										value={ attributes.minutesLeft }
+										attribute="minutesLeft"
+									/>
+									<AdvancedRangeControl
+										label={ __( 'Seconds', i18n ) }
+										min={ 0 }
+										max={ 59 }
+										value={ attributes.secondsLeft }
+										attribute="secondsLeft"
+									/>
+									<AdvancedRangeControl
+										label={ __( 'Restart Countdown After no. of Hours', i18n ) }
+										min={ 0 }
+										maxSlider={ 10 }
+										value={ attributes.restartInterval }
+										attribute="restartInterval"
+									/>
+								</Fragment>
+							) }
+							<AdvancedToggleControl
+								label={ __( 'Enable Double Digit', i18n ) }
```

Comment:
I recommend using this label instead, because if you have "Enable Double Digit", we will be doing in tutorials that they should "turn on the enable double digit option". If it just says "Double digits", then it's just "turn on double digits"

```suggestion
								label={ __( 'Double Digits', i18n ) }
```